packages/trader/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -83,11 +83,12 @@ class AddressDetails extends Component {
                                 </p>
                                 <div className='details-form__elements-container'>
                                     <ThemedScrollbars
+                                        autoHide
                                         style={{
                                             height: '100%',
                                         }}
                                     >
-                                        <div className='details-form__elements'>
+                                        <div className='details-form__elements details-form__elements--overflow'>
```

Comment:
Do we need a modifier class for this? Since there is no condition for not having an overflow, I suggest we use the styles in the `details-form__elaments` class.